.github/workflows/neko.yml

```
@@ -0,0 +1,16 @@
+name: Qiskit Neko Integration Tests
+on:
+  push:
+  pull_request:
+    branches: ['main']
```

Same question from Terra - do we want to run this on PRs to stable branches as well?

---

Yeah, this should include stable too

```suggestion
  push:
  pull_request:
    branches: ['main', 'stable/*']
```